src/components-examples/material/card/card-footer/card-footer-example.html

```diff
@@ -0,0 +1,18 @@
+<mat-card class="example-card">
+  <mat-card-subtitle>Dog Breed</mat-card-subtitle>
+  <mat-card-title>Shiba Inu</mat-card-title>
+  <mat-card-content>
+    <p>This card indeterminates progress bar and divider.</p>
```

Comment:
This says that the card has a progress bar and a divider, but they're both commented out.